### DISCUSSION TITLE
Hacked and Suspended Account ! No response from Github Support after 72 Days !

### DISCUSSION THREAD

**Discussion by asmawi117** [community/community#121035] [Other Feature Feedback, Questions, & Ideas]
**yess, no response from github support team after 72 days my ticket still open**
![51days](https://github.com/community/community/assets/56465197/3476696c-2eef-43d8-a5c1-4c4e919cc1e8)

**link to my hacked and suspended account : https://github.com/youpepelo before it was  https://github.com/asmawilubis**

**the account still connected on my phone but i cant do anything :** 
![GITHUB](https://github.com/community/community/assets/56465197/db823cf1-67a8-4bd4-ad0c-429c5e765fce)

**Here's my ticket : #2593578**

**Comment by Akash1134** [comment]
Hi @asmawi117 , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Comment by ebndev** [comment]
Hi @asmawi117,

You have done the necessary steps on your end by submitting a ticket with our Support Team and I can assure you it's in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

I wanted to reiterate from another discussion that this community is managed by GitHub employees, but it is not a support forum. This is a public forum, and the community is unable to assist with nor escalate your ticket and Community Discussions team members will not be able to provide additional help with account related questions.

As part of our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members keep the community tidy by not duplicating posts across multiple categories. As a result, we have locked this and all duplicate posts, and we ask that you no longer cross-post. 

Any repeat and future violations may result in a temporary or indefinite block from the Community. 

Thank you for your understanding.
